web/src/stream_types.ts

```diff
@@ -21,6 +21,7 @@ export type StreamPermissionGroupSetting = z.infer<typeof stream_permission_grou
 
 // These types are taken from the `zerver/lib/types.py`.
 export const stream_schema = z.object({
+    subscriber_count: z.number(),
```

Comment:
Please add things to sorted lists in the right place in the sorted order. 

I guess this one has like 5 things at the bottom that previous folks have added in a non-sorted position, but clearly it was originally sorted and we should all be helping maintain that.

Reply:
Sorted in the many lines in this PR, and also fixed at least this object's complete sorting in a followup commit